lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -118,22 +127,35 @@ def add_antifraud_data(params, options)
         params[:antifraud] = antifraud
       end
 
-      def commit(action, params)
+      def prepare_refund_data(params, authorization, options)
+        params.delete(:purchaseNumber)
+        params[:externalReferenceId] = params.delete(:externalTransactionId)
+        _ , transaction_id = split_authorization(authorization)
+
+        options.update(transaction_id: transaction_id)
+        params[:ruc] = @options[:ruc]
```

Comment:
What's the rationale for passing `ruc` through `@options` instead of using the `options` hash?